recipes/nginx.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Cookbook Name:: formsender
+# Recipe:: nginx
+#
+# Copyright 2015, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'osl-nginx'
```

Comment:
Any time you have an `include_recipe` call in a recipe, you need to have a corresponding `depends` line in `metadata.rb`. So you should add

```ruby
depends 'osl-nginx'
```

to `metadata.rb`